nemo/collections/asr/models/hybrid_rnnt_ctc_models.py

```diff
@@ -415,9 +416,14 @@ def training_step(self, batch, batch_nb):
             }
 
             if compute_wer:
-                self.wer.update(encoded, encoded_len, transcript, transcript_len)
-                _, scores, words = self.wer.compute()
-                self.wer.reset()
+                self._wer.update(
+                    predictions=encoded,
+                    predictions_lengths=encoded_len,
+                    targets=transcript,
```

Comment:
Don't reorder any arg

Reply:
this isn't reordering, just keyword assignment